config/locales/en.yml

```diff
@@ -988,6 +986,11 @@ en:
       created_on: "created on %{datetime}"
       updated_by_on: "updated by %{user} on %{datetime}"
       updated_on: "updated on %{datetime}"
+      parent_without_of: "Subproject"
+      parent_no_longer: "No longer subproject of"
+      text_journal_changed_html: "%{label} changed from %{old} %{linebreak}to %{new}"
+      text_journal_deleted: "%{label} %{old}"
+      text_journal_of: "%{label} %{value}"
```

Comment:
I think these ones (`textjournal_...`) are better next to the similar ones like before. For the `parent_without_of` and `parent_no_longer`, I think it's better here. (it's subjective)